src/pages/Loans/LoansOverview/hooks/use-get-account-health-factor.tsx

```diff
@@ -0,0 +1,86 @@
+import { CurrencyExt } from '@interlay/interbtc-api';
+import { MonetaryAmount } from '@interlay/monetary-js';
+import Big from 'big.js';
+import { useCallback } from 'react';
+
+import { LoanAction } from '@/types/loans';
+import { useGetAccountLoansOverview } from '@/utils/hooks/api/loans/use-get-account-loans-overview';
+import { useGetLoanAssets } from '@/utils/hooks/api/loans/use-get-loan-assets';
+import { useGetPrices } from '@/utils/hooks/api/use-get-prices';
+
+import { calcutateCollateralBorrowedAmountUSD } from '../utils/math';
+
+const calculateHealthFactor = (borrowAmountUSD: Big, collateralAmountUSD: Big) =>
+  borrowAmountUSD.gt(0) ? collateralAmountUSD.div(borrowAmountUSD).toNumber() : Infinity;
+
+const getCurrencyHealthFactor = (borrowAmountUSD?: Big, collateralAmountUSD?: Big): number | undefined => {
```

Comment:
Same as the previous comment, I would remove 'currency' from the name to keep it more clear.